algorithm-prototyping/brute-force-prototype/src/main/java/edu/uga/devdogs/bruteforceprototype/schedule/ScheduleUtil.java

```diff
@@ -68,7 +68,39 @@ public static double computeAverageIdleTime(Schedule schedule) {
      * @return the overall objective score for the schedule
      */
     public static double computeOverallObjective(Schedule schedule, Map<String, Map<String, Double>> distances, double[] weights) {
```

Comment:
Update the javadoc comment for `computeOverallObjective` to describe the normalization and weighted sum behavior. Additionally, indicate the expected size of `weights` (3) and which indices correspond to which objective.